inst/stan/functions/primary_censored_dist_analytical_cdf.stan

```diff
@@ -125,6 +125,81 @@ real primary_censored_lognormal_uniform_lcdf(data real d, real q, array[] real p
   return log_F_Splus;
 }
 
+/**
+  * Compute the log of the integral of the Weibull PDF from 0 to t
+  *
+  * This function is used in the analytical solution for the primary censored
+  * Weibull distribution with uniform primary censoring. It corresponds to the
+  * g(t; λ, k) function described in the analytic solutions document.
+  *
+  * @param t Upper bound of integration
+  * @param shape Shape parameter (k) of the Weibull distribution
+  * @param scale Scale parameter (λ) of the Weibull distribution
+  *
+  * @return Log of g(t; λ, k) = γ(1 + 1/k, (t/λ)^k) / k - (t/λ) * exp(-(t/λ)^k)
```

Comment:
This is not quite right

its either 

```math
g(t; \lambda, k) = \gamma(1 + 1/k, (t/\lambda)^k)
```

or

```math
g(t; \lambda, k) = \frac{1}{k}\gamma(1/k, (t/\lambda)^k) - \frac{t}{\lambda}e^{-(t/\lambda)^k}.
```

Reply:
ha yes as above this has got mixed up in the debugging. with either it doesn't work

Reply:
wait

Reply:
I think I havee only every had the 1/k